Tasks/FtpUploadV2/task.json

```diff
@@ -112,6 +112,15 @@
             "required": true,
             "helpMarkDown": "Upload files to this directory on the remote FTP server."
         },
+        {
+            "name": "forceUtf8",
+            "type": "boolean",
+            "label": "Force UTF8",
```

Comment:
Possible improvement
label: `Enable Unicode support`
help: `Enables UTF-8 support for the FTP connection ('OPTS UTF8 ON')`